ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/ntp.j2

```diff
@@ -23,8 +23,9 @@
 
 | Node | Primary |
 | ---- | ------- |
-{%     for node in ntp_server.nodes | arista.avd.default([]) %}
-{%         if loop.first %}
+{%     set firstEntry = ntp_server.nodes.0 %}
+{%     for node in ntp_server.nodes | arista.avd.default([]) | arista.avd.natural_sort %}
```

Comment:
This `ntp_server.nodes | arista.avd.default([]) | arista.avd.natural_sort` should be shorten with `ntp_server.nodes | arista.avd.natural_sort`